lua/wikis/commons/Widget/Match/Header.lua

```diff
@@ -24,13 +24,15 @@ local OpponentDisplay = Lua.import('Module:OpponentDisplay/Custom')
 ---@class MatchHeaderProps
 ---@field match MatchGroupUtilMatch
 ---@field teamStyle? teamStyle
+---@field variant 'horizontal' | 'vertical'
```

Comment:
I'm irrationally irritated by there being spaces around the pipe whilst elsewhere there is none (`---@field format ('full'|'compact')?`), not a blocker, just mentioning it as something we might want to normalize repo wide at some point